README.md

```diff
@@ -70,6 +70,28 @@ directory the variables from `.env` will be loaded into the environment.
 
 E.g. `foreman run rails server` or `foreman run rails console`.
 
+## Quick Start  
+####Beginner Friendly Tips for New Contributors  
+- After forking the repo, follow the steps above (Bootstrap & Mailtrap)
+- (Install and) connect to Postgres server  
+- With everything properly installed, open the browser (in development environment)
+- The app should be available, with the database loaded with (fake) data.
+- To access all the functionality of the teams app, add yourself as an organizer. 
+    * In the browser: log in with your github account 
+    * In Console:  
+    ```$ user = User.last``` #or ```user = User.find_by(github_handle: "yourgithubhandle") ```  
```

Comment:
The `$` looks as if it has to be run on the terminal. I would remove it (here and in the following lines)